src/internet_identity_interface/src/internet_identity/conversions.rs

```diff
@@ -71,6 +71,52 @@ impl From<AuthnMethodProtection> for DeviceProtection {
     }
 }
 
+impl From<AuthnMethodPurpose> for Purpose {
```

Comment:
nit: maybe add simple tests for these conversions?  They are trivial, but it would be good to have them executed regularly in some tests.